vaccine/healthcheck_ussd.py

```diff
@@ -155,6 +166,10 @@ async def state_start(self):
         self.save_answer(
             "state_preexisting_conditions", data["data"].get("preexisting_condition")
         )
+        self.save_answer(
+            "state_privacy_policy_accepted",
+            data["data"].get("privacy_policy_accepted", False),
```

Comment:
For flow-results, the spec doesn't have booleans, so we usually store these as yes/no strings.